src/component.ts

```diff
@@ -31,20 +32,20 @@ export function updateElement(
   const id = `${component.id}`;
   let element = document.getElementById(id);
   if (element) {
-    children.forEach((child) => {
-      element?.removeChild(child);
-    });
+    while (element.firstChild) {
```

Comment:
I do think this works fine, but `while` is a seldom used keyword on web. I don't think we need to change it here, but I am curious if you 're referencing this from docs or anywhere else.